### PR TITLE
ddtrace 2.0 shim

### PR DESCRIPTION
**Pull request by TonyCTHsu**
**What does this PR do?**

TBD

### CODE REVIEW

File: ddtrace.gemspec [hunk 1]
```diff
@@ -24,52 +24,28 @@ Gem::Specification.new do |spec|
...
-      bin/**/*
-      ext/**/*
-      lib/**/*
-    ]]
-      .select { |fn| File.file?(fn) } # We don't want directories, only files
-      .reject { |fn| fn.end_with?('.so', '.bundle') } # Exclude local profiler binary artifacts
-      .reject { |fn| fn.end_with?('skipped_reason.txt') } # Generated by profiler; should never be distributed
+  # ddtace 2.0 is a shim to the renamed `datadog` gem
+  spec.files = [
+    'lib/ddtrace/version.rb',         # Should the version file be included ?
```
Comment (marcotc) [inline ddtrace.gemspec:37]: > `# Should the version file be included ?`

I think so

File: ddtrace.gemspec [hunk 2]
```diff
@@ -24,52 +24,28 @@ Gem::Specification.new do |spec|
...
-
-  # Used by profiling (and possibly others in the future)
-  # When updating the version here, please also update the version in `native_extension_helpers.rb` (and yes we have a test for it)
-  spec.add_dependency 'libdatadog', '~> 7.0.0.1.0'
-
-  # used for CI visibility product until the next major version
-  spec.add_dependency 'datadog-ci', '~> 0.8.1'
-
-  spec.extensions = ['ext/datadog_profiling_native_extension/extconf.rb', 'ext/datadog_profiling_loader/extconf.rb']
+    Instead of requiring `ddtrace`, you should now require `datadog` in your Gemfile.
```
Comment (marcotc) [inline ddtrace.gemspec:49]: Let's also add here that `ddtrace` will not receive any further updates.

File: ddtrace.gemspec [hunk 3]
```diff
@@ -24,52 +24,28 @@ Gem::Specification.new do |spec|
...
+  # ddtace 2.0 is a shim to the renamed `datadog` gem
+  spec.files = [
+    'lib/ddtrace/version.rb',         # Should the version file be included ?
+    'lib/ddtrace/auto_instrument.rb', # To support auto-instrumentation in Gemfile
+  ]
 
-  spec.executables   = ['ddtracerb', 'ddprofrb']
-  spec.require_paths = ['lib']
+  # For testing purposes, fixing with a prerelease version
+  spec.add_runtime_dependency 'datadog', '= 2.0.0.beta1'
```
Comment (delner) [inline ddtrace.gemspec:42]: Does this change to a regular dependency before we merge this?

**Comment by ivoanjo** [comment]
Looking at the diff on this one, I believe this effort was abandoned, so I'm going to go ahead and close this one.

**Pull request closed without merging**